cc/BUILD

```diff
@@ -7,7 +7,7 @@ docker_build(
     name = "cc",
     base = "//base:base",
     debs = [
-        "@libgcc1//file:pkg.deb",
-        "@libstdcpp6//file",
+        "@package_bundle//file:libgcc1.deb",
+        "@package_bundle//file:libstdc__6.deb",
```

Comment:
I thought of another trick for how to handle invalid characters over the weekend, it is relevant here.

within `@package_bundle` generate a `.bzl` file, then you:
```python
load("@package_bundle//file:the.bzl", "packages")
```

Where `packages` is a dictionary of unescaped package names to `BUILD` target, then you replace this line with:
```python
   packages["libstdc++6"],
```

... and voila :)

This enables us to employ `base64` encoding or `sha256` to better avoid collisions.

@alexeagle has been battling something similar, but I don't believe this completely solves his problem.